git/Contributing.md

```diff
@@ -0,0 +1,58 @@
+# Contributing
+
+In this file I explained best practice for contributing on a Git version control repository hosting service such as _Github_, _Gitlab_, _Bitbucket_, etc.
+We're going to explain you flow when you're develop application for production.
+
+In flow given in [here](http://nvie.com/posts/a-successful-git-branching-model/) you can see detailed explanation of proposed flow, but we're trying to keep it short.
```

Comment:
Prefer different way of referencing, like the name of the source, like you used in this example
`As you can see in [Erlang's guide](...)`